src/xcode/ENA/ENA/Resources/Storyboards/Settings.storyboard

```diff
@@ -144,16 +144,16 @@
                                 </constraints>
                             </scrollView>
                             <visualEffectView opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="VZq-Fq-6XF">
-                                <rect key="frame" x="0.0" y="668" width="414" height="174"/>
+                                <rect key="frame" x="0.0" y="704" width="414" height="138"/>
                                 <view key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" id="GDV-z5-gaV">
-                                    <rect key="frame" x="0.0" y="0.0" width="414" height="174"/>
+                                    <rect key="frame" x="0.0" y="0.0" width="414" height="138"/>
                                     <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                                     <subviews>
                                         <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="8" translatesAutoresizingMaskIntoConstraints="NO" id="Rfr-Aw-XC1">
-                                            <rect key="frame" x="16" y="16" width="382" height="108"/>
+                                            <rect key="frame" x="16" y="16" width="382" height="72"/>
                                             <subviews>
                                                 <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="wordWrap" translatesAutoresizingMaskIntoConstraints="NO" id="wkA-Dj-yfV" customClass="ENAButton" customModule="ENA" customModuleProvider="target">
-                                                    <rect key="frame" x="0.0" y="0.0" width="382" height="50"/>
+                                                    <rect key="frame" x="0.0" y="0.0" width="382" height="34"/>
```

Comment:
please try to avoid non required frame changes in your commits. Only causes merge conflicts.

Reply:
Ach, dammit Xcode. I needed to add the IBOutlet, and of course Xcode made tons of micro changes. Will update.